couchbase/src/main/scala/akka/stream/alpakka/couchbase/scaladsl/CouchbaseSession.scala

```diff
@@ -53,6 +48,26 @@ object CouchbaseSession {
   def apply(bucket: Bucket): CouchbaseSession =
     new CouchbaseSessionImpl(bucket.async(), None)
 
+  /**
+   * Connects to a Couchbase cluster by creating an `AsyncCluster`.
+   * The life-cycle of it is the user's responsibility.
+   */
+  def createClusterClient(settings: CouchbaseSessionSettings)(implicit ec: ExecutionContext): Future[AsyncCluster] =
+    //wrap CouchbaseAsyncCluster.create up in the Future because it's blocking
+    Future(settings.environment match {
+      case Some(environment) =>
+        CouchbaseAsyncCluster.create(environment, settings.nodes: _*)
+      case None =>
+        CouchbaseAsyncCluster.create(settings.nodes: _*)
+    }).map(_.authenticate(settings.username, settings.password))
```

Comment:
Improvement on the pre-existing logic: since it is blocking, run the future on the blocking dispatcher rather than the default one?